### PR TITLE
Update Documentation

### PR DESCRIPTION
**Pull request by kacieh80**

### CODE REVIEW

File: Splunk Phantom/DomainTools_Guided_Pivots/README.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
+With the  `domaintoolsiriscreds` asset loaded in Phantom, create a custom list named `iris_malicious_tags`.  In this directory there is a csv called iris_malicious_tags.csv. Copy the contents of this into the custom list. The values in the list can be changed to any tags you wish to have flagged.
 
 #### Installation 
-After creating the custom list mentioned above, download the tar file in this directory and import the PB using that file. The assets in PB are for your DomainTools credentials. Point the asset to your dt credentials asset, save the PB and give it a shot.
\ No newline at end of file
+Download the tar file in this directory and import the playbook using that file. The assets in playbook are for your DomainTools credentials. Point the asset to your DomainTools `domaintoolsiriscreds` asset, save the playbook, and give it a shot.
```
Comment (mccarthym) [inline Splunk Phantom/DomainTools_Guided_Pivots/README.md]: Are these changes intended? The playbook in questions _appears_ to be our `DomainTools_Guided_Pivots`, but the content seems to align with our Malicious Tags Playbook.

File: Splunk Phantom/DomainTools_Iris_Malicious_Tags/README.md [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
+With the  `domaintoolsiriscreds` asset loaded in Phantom, create a custom list named `iris_malicious_tags`. in this directory there is a csv called iris_malicious_tags.csv. You will need to copy the contents of this into a custom list on your instance with the same name. The values in the list can be changed to any tags you wish to have flagged.
 
 #### Installation 
-After creating the custom list mentioned above, download the tar file in this directory and import the PB using that file. The assets in PB are for your DomainTools credentials. Point the asset to your dt credentials asset, save the PB and give it a shot.
\ No newline at end of file
+Download the tar file in this directory and import the playbook using that file. The assets in playbook are for your DomainTools credentials. Point the asset to your DomainTools `domaintoolsiriscreds` asset, change the manual response block to send a prompt to the desired role or user, and give it a shot.
```
Comment (mccarthym) [inline Splunk Phantom/DomainTools_Iris_Malicious_Tags/README.md:14]: Should "The assets in playbook.." be "The assets in _the_ playbook .."

File: Splunk Phantom/README.md [hunk 3]
```diff
@@ -1,25 +1,33 @@
...
+
+
 ## DomainTools Phantom Playbooks
 
-Playbooks and automation scripts for Splunk Phantom.
+Working playbooks and automation scripts for Splunk Phantom. 
 
-### Current Playbooks In This Repo
+#### Installation
+For the DomainTools playbooks in this repo, an asset called `domaintoolsiriscreds` is needed with your DomainTools username and key. The playbooks expect this asset to exist to make the API calls needed for the playbook data. Installation instructions for each playbook in the README at the root of the playbook’s directory.
```
Comment (mccarthym) [inline Splunk Phantom/README.md:9]: Should "instructions for each playbook in .." be "instructions for each playbook _is_ in .."

**Review by dnunes-domaintools — APPROVED** [review]

File: Splunk Phantom/DomainTools_Domain_Risk_Score/README.md [hunk 4]
```diff
@@ -1,9 +1,14 @@
...
+Retrieve the Domain Risk Score and throw an Alert for the Analyst to manually review the domain artifact. Users can block the domain.
 
-#### Before Installing
-In this directory there is a csv called domaintools_domain_risk_score.csv. You will need to copy the contents of this into a custom list on your instance with the same name. The name in the first column are used by custom code on the decision block in the PB. The second column is the minimum risk score you would like to alert on.
+#### Needed Asset in Phantom
+The DomainTools playbooks in this repo will require an asset called `domaintoolsiriscreds` with DomainTools API username and key. It is needed to retrieve DomainTools data via the DomainTools API within the playbook.
+<br />
+
+#### Before Installing 
+With the  `domaintoolsiriscreds` asset loaded in Phantom, create a custom list named `domaintools_domain_risk_score`. In this directory there is a csv called domaintools_domain_risk_score.csv. Copy the contents of this into the custom list. The names in the first column are used by custom code on the decision block in the playbook. The second column is the minimum risk score to alert on.
```
[the remark this answers is not in the document]
Comment (kacieh80) [reply Splunk Phantom/DomainTools_Domain_Risk_Score/README.md:10]: @mccarthym I will look into that, I just loaded 2.0 branch into staging

File: Splunk Phantom/README.md (hunk 3 above)
Comment (kacieh80) [reply Splunk Phantom/README.md:9]: Yes

File: Splunk Phantom/README.md [hunk 5]
```diff
@@ -1,25 +1,33 @@
...
 
 [User Prompt and Block Domains](https://github.com/phantomcyber/playbooks/blob/4.2/user_prompt_and_block_domain.py)
 
-This playbook utilizes DomainTools to generate the risk level of a domain, and, if a high enough score, blocks the domain on OpenDNS Umbrella for 60 minutes, after approval via user prompt of a role.
+Utilize DomainTools to generate the risk level of a domain, and, if a high enough score, block the domain on OpenDNS Umbrella for 60 minutes, after approval via user prompt of a role.
 
 [Phishing Investigate & Response](https://github.com/phantomcyber/playbooks/blob/4.2/phishing_investigate_and_respond.py)
  
-This playbook uses various 3rdf party services to investigates and remediates phishing emails with Admin approval 
+Use various 3rdf party services to investigate and remediate phishing emails with Admin approval
```
[the remark this answers is not in the document]
Comment (kacieh80) [reply Splunk Phantom/README.md:32]: Maybe, this is copy paste from Sourin, but it does look like a typo, I will fix it

File: Splunk Phantom/DomainTools_Iris_Malicious_Tags/README.md [hunk 6]
```diff
@@ -1,9 +1,14 @@
 ## DomainTools Iris Malicious Tags Playbook
 
-This playbook retrieves the Iris Investigate profile of a domain and automatically flags domains that have been tagged as malicious in DomainTools Iris. The list of malicious tags are configurable by the analyst and can be kept up to date for automation.
+Retrieves the Iris Investigate profile of a domain and automatically flag domains that have been tagged as malicious in DomainTools Iris. The list of malicious tags are configurable by the analyst and can be kept up to date for automation.
```
[the remark this answers is not in the document]
Comment (kacieh80) [reply Splunk Phantom/DomainTools_Iris_Malicious_Tags/README.md:3]: I moved these to a google doc so that I could get spell check. I really messed them up, I'll fix it. 
Thank you for looking at this @mccarthym, a lot of good catches! I'll let you know when I get it fixed up. Wow me.

[the remark this answers is not in the document]
Comment (kacieh80) [reply Splunk Phantom/DomainTools_Iris_Malicious_Tags/README.md:3]: This file has been fixed
